CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+discussion, or any other method with the owners of this repository before making a change. 
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure that you have signed the CLA: in a separate PR, add your information to 
+   the [Notices](./Notices.md), and [CLA Bot](./.clabot).
+2. Update the README.md with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
```

Comment:
Uh, this point reads weirdly in a spec repo, although generally NBD I think.

Reply:
Yeah. This is the spec standards verbiage directly 🤷‍♀️ I'll update for our use case 👍

Reply:
I just... removed that line